test/test-publication.bash

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
It might be good to split the rename (and shebang line) into a separate PR, to make it easier to see changes over time.